source/sgp_mode/SquareTest.h

```diff
@@ -0,0 +1,40 @@
+#ifndef PHENOTYPE_ANALYSIS_H
```

Comment:
This file isn't only testing squares, so it should be named something more general, like SymbiontImpact.h